src/Submissions/SubmissionsApi.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Smartling\Submissions;
+
+use Psr\Log\LoggerInterface;
+use Smartling\AuthApi\AuthApiInterface;
+use Smartling\BaseApiAbstract;
+use Smartling\Exceptions\SmartlingApiException;
+use Smartling\Submissions\Params\SearchSubmissionsParams;
+
+/**
+ * Class SubmissionsApi
+ * @package Smartling\Submissions
+ */
+class SubmissionsApi extends BaseApiAbstract
+{
+    const ENDPOINT_URL = 'https://api.smartling.com/submissions-api/v2/projects';
+
+    /**
+     * @param AuthApiInterface $authProvider
+     * @param string $projectId
+     * @param LoggerInterface $logger
+     *
+     * @return SubmissionsApi
+     */
+    public static function create(AuthApiInterface $authProvider, $projectId, $logger = null)
+    {
+        $client = self::initializeHttpClient(self::ENDPOINT_URL);
+
+        $instance = new self($projectId, $client, $logger, self::ENDPOINT_URL);
+        $instance->setAuth($authProvider);
+
+        return $instance;
+    }
+
+    /**
+     * @param string $bucketName
+     * @param array $params
+     * @return mixed
+     * @throws SmartlingApiException
+     */
+    public function createSubmission($bucketName, array $params)
```

Comment:
I don't see real need for that because of complexity of Submission entity that may have undefined count of details inside.
Also real logic will have next steps: `connector entity` -> `array` -> `SubmissionParams constructor` -> `exported array`.
All checks of related submissions and details are implemented in submission service, so skipping `SubmissionParams` object makes chain easier.

Reply:
Ok, why do you see real need in `SearchSubmissionsParams` then?)

Reply:
Because it has logic for json fields and sone checks

Reply:
> I don't see real need for that because of complexity of Submission entity that may have undefined count of details inside.

So there would be `$submission_create_params->addDetail($detail)`. Let's be consistent.

Reply:
Added params for creation and update